app/services/album.js

```diff
@@ -3,16 +3,29 @@ const axios = require('axios'),
   base_uri = config.common.albumsApi.url,
   logger = require('../logger');
 
-exports.getAlbum = params =>
+exports.allAlbums = (offset, limit, orderedBy) =>
   axios
-    .get(`${base_uri}/albums/${params.id}`)
+    .get(`${base_uri}/albums`)
+    .then(response => {
+      const pagination = response.data.slice(offset, offset + limit);
```

Comment:
I would abstract this functionality (order, filter, etc). Maybe could be useful in other resolvers